src/main.py

```diff
@@ -33,6 +33,16 @@ def main():
     neuron_build_mode = parse_dlc_developer_configs("dev", "neuron_mode")
     graviton_build_mode = parse_dlc_developer_configs("dev", "graviton_mode")
 
+    # Write empty dict to JSON file, so subsequent buildspec steps do not fail in case we skip this build
+    utils.write_to_json_file(constants.TEST_TYPE_IMAGES_PATH, {})
+
+    # Add ability to skip tensorflow-1 or tensorflow-2
+    build_arn = utils.get_codebuild_build_arn()
+    if "tensorflow-1" in build_arn and "tensorflow-1" in frameworks_to_skip:
```

Comment:
do we need to check the `build_context` here?

Reply:
yeah we can do that